inference-engine/ie_bridges/java/org/intel/openvino/IECore.java

```diff
@@ -64,11 +66,13 @@ public Parameter GetConfig(String deviceName, String name) {
     /*----------------------------------- native methods -----------------------------------*/
     private static native long ReadNetwork(long core, final String modelFileName);
 
-    private static native long ReadNetwork1(long core, final String modelPath, final String weightPath);
+    private static native long ReadNetwork1(
+            long core, final String modelPath, final String weightPath);
```

Comment:
Same here and everywhere with more than 80 symbols line length. Try to fit this limit:
```java
    private static native long ReadNetwork1(long core, final String modelPath,
                                            final String weightPath);
```

Reply:
It causes code style error.